netbox/netbox/settings.py

```diff
@@ -859,6 +859,14 @@ def _setting(name, default=None):
         f"{plugin_name}.{queue}": RQ_PARAMS for queue in plugin_config.queues
     })
 
+    events_pipeline = plugin_config.events_pipeline
+    if events_pipeline and type(events_pipeline) in (list, tuple):
+        EVENTS_PIPELINE = list(EVENTS_PIPELINE)
```

Comment:
Constants should never be redefined. Any validation of the initial `EVENTS_PIPELINE` config parameter should be performed prior to plugin registration.

Reply:
? We are doing that all over settings.py, just a few lines above we are doing the same to INSTALLED_APPS.  Previously this was assigned as a tuple and could be in config.py, so that is why had it convert to a list.